deployment/chainloop/values.yaml

```diff
@@ -182,6 +188,14 @@ controlplane:
       clientID: ""
       clientSecret: ""
 
+    ## @extra controlplane.auth.allowList.rules List of domains or emails to allow
+    ## @extra controlplane.auth.allowList.selectedRoutes List of selected routes to allow
```

Comment:
indicate that if not set means that it applies to all routes

Reply:
Done.